plugins/woocommerce/phpcs.xml

```diff
@@ -102,4 +102,24 @@
 	<rule ref="Squiz.Commenting.FileComment.Missing">
 		<exclude-pattern>tests/php/</exclude-pattern>
 	</rule>
+
+	<rule ref="WordPress.NamingConventions.ValidVariableName">
+		<exclude-pattern>tests/php/src</exclude-pattern>
+	</rule>
+
+	<rule ref="Squiz.Commenting">
+		<exclude-pattern>tests/php/src</exclude-pattern>
+	</rule>
+
+	<rule ref="WordPress.WP.GlobalVariablesOverride">
+		<exclude-pattern>tests/php/src</exclude-pattern>
+	</rule>
+
+	<rule ref="Generic.Arrays.DisallowShortArraySyntax">
+		<exclude-pattern>tests/php/src</exclude-pattern>
```

Comment:
There's been a lot of discussion already so apologies if I missed a previous answer, but for some of these exclusions, and this one (`[]` vs `array()` syntax) in particular ... why is the exclusion required/desired?

Tests are still a part of the codebase, and the reason for rules like this one is readability (whether or not we agree with it as individuals, it's something we inherit from the core WP coding standard and probably shouldn't veer from ... unless perhaps WP Core make the same exception for their tests, but I [don't think they do](https://github.com/WordPress/wordpress-develop/blob/trunk/phpcs.xml.dist)).

Reply:
I think this was something that was copied from the time this was a separate plugin. We can clear that up and conform to the standards used by WooCommerce. CC @layoutd

Reply:
@budzanowski looks right, probably copied over from woocommerce/woocommerce-order-source-attribution, I'll remove these new exclusions (fc2301ec357ef5e42b318ebddbb078d999b1df23).